demos/classification_demo/grid_mat.hpp

```diff
@@ -29,6 +31,7 @@ struct LabeledImage {
 class GridMat {
 public:
     cv::Mat outImg;
+    PerformanceMetrics performanceMetrics;
```

Comment:
Should this really be here? The performance metrics don't really have anything to do with grid-ness; maybe it would be better to have it as local variable of `main` instead?

Reply:
Moved